src/serac/infrastructure/accelerator.hpp

```diff
@@ -72,6 +72,24 @@ enum class ExecutionSpace
   Dynamic  // Corresponds to execution that can "legally" happen on either the host or device
 };
 
+#ifdef SERAC_USE_CUDA_KERNEL_EVALUATION
+
+/// @brief Alias for parallel threads policy on GPU
+using threads_x     = RAJA::LoopPolicy<RAJA::cuda_thread_x_direct>;
+using teams_e       = RAJA::LoopPolicy<RAJA::cuda_block_x_direct>;
```

Comment:
Should we use a more descriptive name than `x` and `e`?

Reply:
It would be better to not have to guess.